### PR TITLE
Switch to calling Files.newBufferedReader

### PR DESCRIPTION
**Pull request by ghost**
These changes were suggested by Extra Hints for NetBeans IDE:
http://plugins.netbeans.org/plugin/73447/

This is somewhat related to [JENKINS-42934](https://issues.jenkins-ci.org/browse/JENKINS-42934).

### Proposed changelog entries

None required

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

File: core/src/main/java/hudson/Util.java
Comment (oleg-nenashev) [inline core/src/main/java/hudson/Util.java]: would be nice to use `Util.fileToPath` to avoid Runtime exceptions. But it can be done in a separate PR

**Review by oleg-nenashev — APPROVED** [review]

File: core/src/main/java/hudson/model/Queue.java
Comment (daniel-beck) [inline core/src/main/java/hudson/model/Queue.java]: I wonder why we're not using UTF-8 here.

**Review by daniel-beck — APPROVED** [review]

File: core/src/main/java/hudson/model/Queue.java
Comment (oleg-nenashev) [reply core/src/main/java/hudson/model/Queue.java]: dunno. I am fine with fixing it in a separate PR. So far I think that all admins running Jenkins with non-UTF-8 just like to live dangerously, so I just wish them good luck

**Comment by oleg-nenashev** [comment]
Thanks @dtrebbien !

File: core/src/main/java/hudson/Util.java
Comment (dtrebbien) [reply core/src/main/java/hudson/Util.java]: I created PR #3219 to use fileToPath().

File: core/src/main/java/hudson/model/Queue.java
Comment (dtrebbien) [reply core/src/main/java/hudson/model/Queue.java]: I will submit a PR to make this change. Because it will conflict with PR #3219, I will wait until PR #3219 is merged before submitting a PR to change the charset.

Comment (dtrebbien) [reply core/src/main/java/hudson/model/Queue.java]: I created PR #3224 to switch to UTF-8.

**Pull request merged**
